Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -78,35 +73,34 @@ - (BOOL)managedObjectContext:(NSManagedObjectContext *)context
                  originalImp:(BOOL(NS_NOESCAPE ^)(NSError **))original
 {
 
-    __block id<SentrySpan> fetchSpan = nil;
+    __block id<SentrySpan> saveSpan = nil;
     if (context.hasChanges) {
         __block NSDictionary<NSString *, NSDictionary *> *operations =
             [self groupEntitiesOperations:context];
 
         [SentrySDK.currentHub.scope useSpan:^(id<SentrySpan> _Nullable span) {
-            fetchSpan = [span startChildWithOperation:SENTRY_COREDATA_SAVE_OPERATION
-                                          description:[self descriptionForOperations:operations
-                                                                           inContext:context]];
-            fetchSpan.origin = SentryTraceOriginAutoDBCoreData;
-            if (fetchSpan) {
-                [SentryLog
-                    logWithMessage:[NSString
-                                       stringWithFormat:@"SentryCoreDataTracker automatically "
-                                                        @"started a new span with description: %@, "
-                                                        @"operation: %@",
-                                       fetchSpan.description, SENTRY_COREDATA_FETCH_OPERATION]
```

Comment:
This should've been SENTRY_COREDATA_SAVE_OPERATION, but I did one better and just access SentrySpan.operation for both such logs (here and in fetches)